src/Bedrock.Framework.Experimental/Protocols/Http1ResponseMessageReader.cs

```diff
@@ -59,7 +59,7 @@ public bool TryParseMessage(in ReadOnlySequence<byte> input, ref SequencePositio
                     goto case State.Headers;
 
                 case State.Headers:
-                    while (sequenceReader.TryReadTo(out var headerLine, NewLine))
+                    while (sequenceReader.TryReadTo(out ReadOnlySequence<byte> headerLine, NewLine))
```

Comment:
Undo

Reply:
On .NET 6 you get the following error when you use the `var`:
```
Severity	Code	Description	Project	File	Line	Suppression State
Error	CS0121	The call is ambiguous between the following methods or properties: 'SequenceReader<T>.TryReadTo(out ReadOnlySequence<T>, ReadOnlySpan<T>, bool)' and 'SequenceReader<T>.TryReadTo(out ReadOnlySpan<T>, ReadOnlySpan<T>, bool)'	Bedrock.Framework.Experimental	c:\Users\radocz.adam\source\repos\Bedrock\src\Bedrock.Framework.Experimental\Protocols\Http1RequestMessageReader.cs	57	Active
```